### ISSUE TITLE
Add sample for ISupportsConfigureWebHost

### ISSUE THREAD

**Issue by dodyg** [dodyg/practical-aspnetcore#306]

**Comment by Samkituka** [comment]
Hello ,
I would like to work on this issue. What would like me to do exactly as a description to this issue ?

**Comment by dodyg** [comment]
Yup. This will need a bit of research. Create a simple sample demonstrating a single idea.

**Comment by Samkituka** [comment]
> Yup. This will need a bit of research. Create a simple sample demonstrating a single idea.

Hello Dodyg,
What do you think of this one? 
https://github.com/Samkituka/Sample-for-ISupportsConfigureWebHost.git

Let me know if you need some adjustments.
Thanks!
